Externals.cfg

```diff
@@ -29,12 +29,12 @@ required = True
 [mizuRoute]
 local_path = components/mizuRoute
 protocol = git
-repo_url = https://github.com/nmizukami/mizuRoute
-hash = 34723c2
+repo_url = https://github.com/ESCOMP/mizuRoute
+hash = 81c720c
```

Comment:
@ekluzek I am not confident about my mizuRoute changes.

Feel free to check the others as well, though. Thanks.

Reply:
This didn't work, so I will revert it to proceed with testing.
`./manage_externals/checkout_externals` returned this error:
```
ERROR:root:Command '['git', '-C', '/glade/work/slevis/git/latest_master/components/mizuRoute', 'fetch', '--quiet', '--tags', 'ESCOMP_mizuRoute']' returned non-zero exit status 1.
ERROR:root:In directory
    /glade/work/slevis/git/latest_master/components/cmeps
Process did not run successfully; returned status 1:
    git -C /glade/work/slevis/git/latest_master/components/mizuRoute fetch --quiet --tags ESCOMP_mizuRoute

ERROR:root:In directory
    /glade/work/slevis/git/latest_master/components/cmeps
Process did not run successfully; returned status 1:
    git -C /glade/work/slevis/git/latest_master/components/mizuRoute fetch --quiet --tags ESCOMP_mizuRoute


ERROR: In directory
    /glade/work/slevis/git/latest_master/components/cmeps
Process did not run successfully; returned status 1:
    git -C /glade/work/slevis/git/latest_master/components/mizuRoute fetch --quiet --tags ESCOMP_mizuRoute
```

Reply:
For mizuRoute try using the ESCOMP fork for this tag: [cesm-coupling.n02_v2.1.2](https://github.com/ESCOMP/mizuRoute/releases/tag/cesm-coupling.n02_v2.1.2)

Reply:
The error looks the same with that.

Reply:
@slevis-lmwg I'm not understanding how you are getting this command:
`git -C /glade/work/slevis/git/latest_master/components/mizuRoute fetch --quiet --tags ESCOMP_mizuRoute`
You have ESCOMP_mizuRoute where I would expect to see https://github.com/ESCOMP/mizuRoute

Reply:
Thank you for checking, @jedwards4b. 
So for now I will keep the old settings for mizuRoute, since they work.

Reply:
Note for me and Erik: 
I have not pushed the old settings back to the PR until I have looked at this with @ekluzek.

Reply:
Since @slevis-lmwg is our traveling, I took a look at this and pushed a simple change that fixes this issue. This has CTSM using the same version as CESM is pointing to. So I'm resolving.

Reply:
Thank you @ekluzek